src/main/webapp/less/includes/_base.less

```
@@ -178,4 +178,15 @@
     .no-col-padding {
         padding: 0;
     }
+
+    div.member-users {
```

is div needed for specificity?
could it be removed?
```suggestion
    .member-users {
```